### DISCUSSION TITLE
JDBC(Star durst) connectivity issue Using Splunk DB Connect

### DISCUSSION THREAD

**Discussion by rsathish4433** [trinodb/trino#17959] [Q&A]
Hello Team,

I am trying to connect statdurst  using JDBC Driver in Splunk DB connect app and getting error 
It looks like am missing to set time zone parament in the connection sting . Could you please help me to find correct parameter to sent the time zone .

Error:
Error executing query: The datetime zone id 'GMT+10:00' is not recognised

Trino JDBC driver version .417

JDBC URL:
jdbc:trino://<<hostname>>:443?SSL=True&SSLVerification=NONE

please let me know if you need any more details.

**Comment by hashhar** [comment]
For questions regarding Starburst reach out to Starburst support or the #vendor-starburst channel on [Trino slack](https://trino.io/slack.html).
